### PR TITLE
Disable `var/` in proc args lint

### PR DESCRIPTION
**Pull request by ike709**
## About The Pull Request

Maintainers decided to disable it.

## Changelog
:cl:
code: Coders will no longer see warnings for using "var/" in proc args. Including it is encouraged.
/:cl:

### CODE REVIEW

**Comment by ike709** [comment]
Am I blind? What's the CI failing on?

**Comment by KubeRoot** [comment]
> Am I blind? What's the CI failing on?

I had the same issue trying to disable it on my side - if you try running it from the language server client, you might be able to find the error. The actual "diagnostic" it's giving is actually an error when parsing the config, I couldn't figure out how to make the config work correctly.

File: SpacemanDMM.toml [hunk 1]
```diff
@@ -5,5 +5,8 @@ dreamchecker = true
 disallow_relative_type_definitions = true
 disallow_relative_proc_definitions = true
 
+[diagnostics]
+var_in_proc_parameter = false
```
Comment (Crossedfall) [inline SpacemanDMM.toml]:
```suggestion
var_in_proc_parameter = "off"
```

It's expecting an error type, which is a string.

Comment (Crossedfall) [reply SpacemanDMM.toml]: https://github.com/SpaceManiac/SpacemanDMM/blob/8c9b6b19c1d68b23d6f1c4248dee83b7546da2cb/crates/dreammaker/src/parser.rs#L1054

Comment (ike709) [reply SpacemanDMM.toml]: Well that's misleading: 
image

**Comment by utilbird** [comment]
sad

**Pull request merged**